cache/cacher.go

```diff
@@ -0,0 +1,31 @@
+package cache
+
+import "fmt"
+
+type Cacher interface {
+	Set(string, []byte) error
+	Get(string) ([]byte, error)
+}
+
+func NewMockCacher() Cacher {
+	return exampleWithNoPersistence{
+		kv: make(map[string][]byte),
+	}
+}
+
+type exampleWithNoPersistence struct {
+	kv map[string][]byte
+}
+
+func (e exampleWithNoPersistence) Set(key string, value []byte) error {
+	e.kv[key] = value
+	return nil
```

Comment:
Returning an `error` despite not needing one here is intentional so another implementation could error if it needed to.